internal/static/i18n/en.yaml

```
@@ -53,11 +53,24 @@ Errors:
     NotFound: SMS configuration not found
     AlreadyActive: SMS configuration already active
     AlreadyDeactivated: SMS configuration already deactivated
+  SMTP:
+    NotEmailMessage: message is not EmailMessage
+    RequiredAttributes: subject, recipients and content must be set but some or all of them are empty
+    CouldNotSplit: could not split host and port for connect to smtp
+    CouldNotDial: could not contact with the SMTP server, check the port, firewall issues...
+    CouldNotDialTLS: could not contact with the SMTP server using TLS, check the port, firewall issues...
+    CouldNotCreateClient: could not create smtp client
+    CouldNotStartTLS: could not start tls
+    CouldNotAuth: could not add smtp auth, check if both your user and password are correct, if they're correct maybe your provider requires an auth method not supported by Zitadel
```

```suggestion
    CouldNotAuth: could not add smtp auth, check if both your user and password are correct, if they're correct maybe your provider requires an auth method not supported by ZITADEL
```

Can you correct this in the other files too, please?